long-ride-alerts/src/main/java/org/apache/flink/training/exercises/longrides/LongRidesExercise.java

```diff
@@ -98,17 +99,56 @@ public static void main(String[] args) throws Exception {
     @VisibleForTesting
     public static class AlertFunction extends KeyedProcessFunction<Long, TaxiRide, Long> {
 
+        private ValueState<TaxiRide> rideState;
+        private Duration TIME_FOR_ALERT = Duration.ofHours(2);
```

Comment:
name breaks Java naming convention, it's not Constants, so it cannot be in uppercase, pls fix it